src/vs/workbench/contrib/testing/browser/testingDecorations.ts

```diff
@@ -318,47 +318,11 @@ export class TestingDecorationService extends Disposable implements ITestingDeco
 				}
 			}
 
-			const lastResult = this.results.results[0];
-			if (this.testService.showInlineOutput.value && lastResult instanceof LiveTestResult) {
-				for (const task of lastResult.tasks) {
-					for (const m of task.otherMessages) {
-						if (!this.invalidatedMessages.has(m) && m.location?.uri.toString() === uriStr) {
-							const decoration = lastDecorations.getMessage(m) || this.instantiationService.createInstance(TestMessageDecoration, m, undefined, model);
-							newDecorations.addMessage(decoration);
-						}
-					}
-				}
-
-				const messageLines = new Map</* line number */ number, /* last test message */ ITestMessage>();
-				for (const test of lastResult.tests) {
-					for (let taskId = 0; taskId < test.tasks.length; taskId++) {
-						const state = test.tasks[taskId];
-						for (let i = 0; i < state.messages.length; i++) {
-							const m = state.messages[i];
-							if (this.invalidatedMessages.has(m) || m.location?.uri.toString() !== uriStr) {
-								continue;
-							}
-
-							// Only add one message per line number. Overlapping messages
-							// don't appear well, and the peek will show all of them (#134129)
-							const line = m.location.range.startLineNumber;
-							if (messageLines.has(line)) {
-								newDecorations.removeMessage(messageLines.get(line)!);
-							}
-
-							const decoration = lastDecorations.getMessage(m) || this.instantiationService.createInstance(TestMessageDecoration, m, buildTestUri({
-								type: TestUriType.ResultActualOutput,
-								messageIndex: i,
-								taskIndex: taskId,
-								resultId: lastResult.id,
-								testExtId: test.item.extId,
-							}), model);
-
-							newDecorations.addMessage(decoration);
-							messageLines.set(line, decoration.testMessage);
-						}
-					}
-				}
+			const messageLines = new Map</* line number */ number, /* last test message */ ITestMessage>();
```

Comment:
This could now just be a `Set<number>` since the value is no longer used

Reply:
Good idea, I'll update it.e